static/admin/config.yml

```diff
@@ -258,6 +258,69 @@ collections:
                         },
                     ],
             }
+          - {
+                label: "Editing Design",
+                name: "editing_design",
+                widget: "object",
+                required: false,
+                collapsed: true,
+                fields:
+                    [
+                        {
+                            label: "cRNA Target Site",
+                            name: "crna_target_site",
+                            widget: "string",
+                        },
+                        {
+                            label: "DNA Donor Sequence",
+                            name: "dna_donor_sequence",
+                            widget: "string",
+                        },
+                        { label: "Cas9", name: "cas9", widget: "string" },
+                        {
+                            label: "F Primer for PCR/Sequencing",
+                            name: "f_primer",
+                            widget: "string",
+                        },
+                        {
+                            label: "R primer for PCR/Sequencing",
+                            name: "r_primer",
+                            widget: "string",
+                        },
+                        {
+                            label: "Diagrams",
+                            name: "diagrams",
+                            widget: "list",
```

Comment:
I think this may need a "required: false" as well. (I haven't tested yet, but if you can't make changes to the rest of the data without having any diagrams that would be a problem)

Reply:
Yeah that's true, I have tested it. We need to add `required: false` to each field(I know)

Reply:
Added, thanks

Reply:
We need to set `required: false` for each field in `Editing Design`, like `cRNA Target Site`, `DNA Donor Sequence`, and others. This will allow us to make changes to the rest of fields without triggering any warnings.

Reply:
Ok, added to those fields.